packages/round-manager/package.json

```diff
@@ -69,39 +64,26 @@
     "react-dom": "^18.1.0",
     "react-dropzone": "^14.2.3",
     "react-gtm-module": "^2.0.11",
-    "react-hook-form": "^7.31.3",
-    "react-icons": "^4.7.1",
-    "react-redux": "^8.0.1",
-    "react-router-dom": "6",
-    "react-scripts": "5.0.1",
+    "react-hook-form": "^7.43.9",
+    "react-icons": "^4.8.0",
+    "react-router-dom": "^6.11.2",
     "react-tooltip": "^4.4.2",
-    "redux": "^4.2.0",
-    "redux-thunk": "^2.4.2",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
-    "swr": "^2.0.0",
-    "typescript": "^4.6.0",
+    "swr": "^2.1.5",
+    "typescript": "^5.0.4",
     "url": "^0.11.0",
     "util": "^0.12.4",
-    "wagmi": "0.6.8",
+    "viem": "^0.3.36",
+    "wagmi": "1.0.6",
     "web-vitals": "^2.1.0",
     "yup": "^0.32.11"
   },
-  "jest": {
-    "transformIgnorePatterns": [
-      "/round-manager/node_modules/(?!(@spruceid))/"
-    ],
-    "moduleNameMapper": {
-      "^csv-stringify/sync": "<rootDir>/node_modules/csv-stringify/dist/cjs/sync.cjs"
-    }
-  },
   "browserslist": {
     "production": [
-      "chrome >= 67",
-      "edge >= 79",
-      "firefox >= 68",
-      "opera >= 54",
-      "safari >= 14"
+      "last 1 chrome version",
```

Comment:
is this safe to do? the `defaults` option seems like a more sensible setting for production, what do you think?

Reply:
good catch, this was a remnant of development, reverted back to defaults.